components/api/src/main/java/com/hotels/styx/api/messages/FullHttpRequest.java

```diff
@@ -366,14 +359,26 @@ public Builder() {
             this.cookies = new ArrayList<>();
         }
 
-        public Builder(HttpMethod method, String uri) {
+        public Builder(String method, String uri) {
```

Comment:
It would be better to introduce a `HttpMethod` type. It would be generally more type safe approach and ensure that only specific values of the `method` are meaningful. Not to mention it would contribute towards more self-commenting code.